storage/memory/peer_store.go

```diff
@@ -125,6 +125,9 @@ func New(cfg Config) (storage.PeerStore, error) {
 	ps.wg.Add(1)
 	go func() {
 		defer ps.wg.Done()
+		if cfg.PrometheusReportingInterval == 0 {
+			cfg.PrometheusReportingInterval = 1
+		}
```

Comment:
I think a warning in the logs (something like "...not specified/invalid, setting to one second") would be best :)